bot/cogs/deployment.py

```diff
@@ -3,7 +3,9 @@
 from discord import Colour, Embed
 from discord.ext.commands import AutoShardedBot, Context, command
 
-from bot.constants import ADMIN_ROLE, DEPLOY_BOT_KEY, DEPLOY_SITE_KEY, DEPLOY_URL, DEVOPS_ROLE, OWNER_ROLE, STATUS_URL
+from bot.constants import (
+    Keys, Roles, URLs
```

Comment:
Is there any point multi-line importing or just 3 things, earlier in this file we import `AutoShardedBot`, `Context` and `command` on a single line